docs/src/limitations.md

```diff
@@ -28,3 +28,13 @@ repeatedly executing code that modifies some global state
 can lead to unexpected side effects.
 Rebugger works best on methods whose behavior is determined solely by their input
 arguments.
+
+## Note for Ubuntu users
```

Comment:
Should this be on the usage or configuration page instead? Currently the limitations page is more about language issues or the algorithmics of Rebugger/Revise, rather than keybindings or user customization.

Reply:
In fact, I believe it should be on the Installation page because these are the information each Ubuntu/Mac/Windows user should see immediately when installing, otherwise they would not be able to follow the examples in the manual. 

Should I create the installation page? It would be a rather short one.

Reply:
Sure, that would be great.

Reply:
I have moved the Ubuntu-specific paragraph to the installation subpage. Sorry, that it took so long.